lib/app/app_constants.dart

```diff
@@ -1,12 +1,12 @@
 class AppConstants {
-  static const String NOW_SHOWING = 'now_showing';
-  static const String POPULAR = 'popular';
-  static const String LENGTH = 'Length';
-  static const String LANGUAGE = 'Language';
-  static const String RATING = 'Rating';
-  static const String DESCRIPTION = 'Description';
-  static const String CASTS = 'Casts';
+  static const String nowShowing = 'now_showing';
```

Comment:
const should always be in uppercase with underscores so it is self descriptive it is const and wont be change.

Reply:
Sure I will make the change and update the request